data/channels.csv

```diff
@@ -3534,9 +3534,11 @@ ChallengeIreland.uk,Challenge Ireland,,UK,,,c/UK,eng,,FALSE,http://www.challenge
 ChallengeUK.uk,Challenge UK,,UK,,,c/UK,eng,,FALSE,http://www.challenge.co.uk/,https://d2n0069hmnqmmx.cloudfront.net/epgdata/1.0/newchanlogos/400/160/skychb2202.png
 ChamberTV.lu,Chamber TV,,LU,,,c/LU,ltz,,FALSE,http://www.chd.lu/,https://i.imgur.com/YldMnap.jpg
 ChampionsTV.ng,Champions TV,,NG,,,c/NG,eng,,FALSE,,https://www.lyngsat.com/logo/tv/cc/champions-tv-ng.png
-ChampionsTV1.id,Champions TV 1,,ID,,,c/ID,ind,,FALSE,,https://www.lyngsat.com/logo/tv/cc/champions-tv-1-id.png
-ChampionsTV2.id,Champions TV 2,,ID,,,c/ID,ind,,FALSE,,https://www.lyngsat.com/logo/tv/cc/champions-tv-2-id.png
-ChampionsTV3.id,Champions TV 3,,ID,,,c/ID,ind,,FALSE,,https://www.lyngsat.com/logo/tv/cc/champions-tv-3-id.png
+ChampionsTV1.id,Champions TV 1,,ID,,,c/ID,ind,,FALSE,https://www.lyngsat-logo.com/logo/tv/cc/champions-tv-1-id.png,https://www.lyngsat.com/logo/tv/cc/champions-tv-1-id.png

```

Comment:
```suggestion
ChampionsTV1.id,Champions TV 1,,ID,,,c/ID,ind,,FALSE,,https://www.lyngsat.com/logo/tv/cc/champions-tv-1-id.png
```

This should be a link to the official website of the channel (https://github.com/iptv-org/database#channels)

The same goes for the rest of the entries.